test/algorithm/Aqua_vancouver.test

```diff
@@ -0,0 +1,92 @@
+                                KMC ACCESS TIME TEST FILE
+====================================================================================================
+Satellite Name: Aqua
+Target Point: 49.07, -123.113
```

Comment:
All XXXX_vancouver.test files are moved to "test/test_data/" in master.
Also, "Target Point" format in the .test files needs to be updated because of the test parser changes.